src/nodes/sq/sq.h

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2025 Team Dissolve and contributors
+
+#pragma once
+
+#include "classes/partialSet.h"
+#include "math/averaging.h"
+#include "math/function1D.h"
+#include "math/windowFunction.h"
+#include "module/module.h"
+#include "nodes/graph.h"
+#include "nodes/node.h"
+#include "nodes/parameter.h"
+
+// Forward Declarations
+class BraggModule;
+class PartialSet;
+
+class SQNode : public Node
+{
+    public:
+    SQNode(Graph *parentGraph);
+    ~SQNode() override = default;
+
+    public:
+    std::string_view type() const override;
+    std::string_view summary() const override;
+
+    /*
+     * Definition
+     */
+    private:
+    // Unweighted g(r)
+    PartialSet *unweightedgr_{nullptr};
+    // Unweighted S(Q)
+    PartialSet unweightedsq_;
```

Comment:
Naming suggestion, here and throughout:
```suggestion
    PartialSet *unweightedGR_{nullptr};
    // Unweighted S(Q)
    PartialSet unweightedSQ_;
```